Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -1013,6 +1013,9 @@ public void ProcessSynchronousEvents(bool forceProcess = false)
                 {
                     foreach (var subscription in _dataFeed.Subscriptions)
                     {
+                        // OI subscription doesn't contain asset market prices
+                        if (subscription.Configuration.TickType == TickType.OpenInterest)
```

Comment:
Perhaps the configuration should expose a flag indicating whether or not it refers to price data.